### DISCUSSION TITLE
Cant solve problem bcs i dont see it

### DISCUSSION THREAD

**Discussion by gudai-a11y** [TCA166/CK3-history-extractor#5] [Q&A] [answered]
Really good project, but using cmd for gui make me angry. Anytime i try to make game history i getting errors and cant read em. How can i fix this problem?

**Comment by TCA166** [comment, marked as answer]
Well you can just launch the game in a pre-existing cmd window, so that it doesn't exit once the program terminates.
If I recall correctly this can be done with the ```start``` command on Windows, or according to [this](https://superuser.com/a/876992) post you can just drag the executable onto the window. I didn't make a GUI because well it's a lot of work, getting multi platform GUI working is tricky and not much point in developing a GUI since it's not like you interact with the program at all after the initial stage. In case of any other issues please do let me know.

**Reply by gudai-a11y** [reply]
Thx man, good day!
